test/abstract_client.js

```diff
@@ -419,6 +419,25 @@ module.exports = function (server, config) {
       });
     });
 
+    it('should emit a publish event', function (done) {
+      var client = connect(),
+        payload = 'test_payload',
+        topic = 'test_topic';
+
+      client.once('publish', function (topic, message, packet) {
+        topic.should.equal(topic);
```

Comment:
Nonetheless, you're right. The topic.should.equal line isn't actually testing anything. I'll adjust that.

I thought I checked this thread earlier but didn't see any comments. Oh well.